rag/easy-rag/runtime/pom.xml

```diff
@@ -22,8 +22,16 @@
                     <groupId>xml-apis</groupId>
                     <artifactId>xml-apis</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>dev.langchain4j</groupId>
+                    <artifactId>langchain4j-embeddings-bge-small-en-v15-q</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
+            <artifactId>langchain4j-embeddings</artifactId>
```

Comment:
Is this necessary? If the app declares a dependency on a particular in-process embedding model, this will be brought in transitively. And if they use a remote embedding model, then this isn't necessary at all, no?